index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="container">
+      <header class="toolbar">
+        <div class="logo">
+          <div class="placeholder">
+            <h3>Eshetu Melaku</h3>
+          </div>
+        </div>
+
+        <div class="normal-button">
+          <div id="icon-menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+          </div>
+        </div>
+      </header>
+    </nav>
+
+    <section class="presentation">
+      <h1 class="title-presentation">Hi There. I'm Melaku</h1>
+      <span>I’am a Software Developer</span>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <footer>
+        <div class="contact">
+          <ul>
+            <li class="icon">
+              <a href="https://twitter.com/home"
+                ><img src="images/twitter.png" alt="linked link"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="https://www.linkedin.com/in/melaku-eshetu-b34b36223//"
+                ><img src="images/Linkedin.png" alt="linked link"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="#" target="_blank"
+                ><img src="images/medium.png" alt=""
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="https://github.com/Melaku05"
+                ><img src="images/github.png" alt="github link"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="#"
+                ><img src="images/angellist.png" alt="Angel list link"
+              /></a>
+            </li>
+          </ul>
```

Comment:
Kindly try to match the color of the `social icon` with the Figma design 
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/154274309-14d996e2-3331-4f40-89df-298bdc74144e.png) | ![image](https://user-images.githubusercontent.com/24703171/154274386-6bc4625e-80f9-429c-9321-787437324836.png) |
You can take it as an image from the Figma design or use an external library for icons like `font awsome`.

Reply:
Thanks again, I just did that also.

Reply:
Addressed 👌